inc/saitunnel.h

```diff
@@ -323,8 +323,30 @@ typedef enum _sai_tunnel_type_t
 
     SAI_TUNNEL_TYPE_MPLS,
 
+    SAI_TUNNEL_TYPE_IPINIP_ESP,
+
+    SAI_TUNNEL_TYPE_IPINIP_UDP_ESP,
+
+    SAI_TUNNEL_TYPE_VXLAN_UDP_ESP,
+
 } sai_tunnel_type_t;
 
+/**
+ * @brief Defines VXLAN tunnel UDP source port mode
+ */
+typedef enum _sai_tunnel_vxlan_udp_sport_mode_t
```

Comment:
have we confirmed if udp sport is needed at 1.7 ?
as far as i understand, we are addressing the dual tor scenario
for which we need the loopback action, and the qos maps

we don't need, as i understand, the vxlan sport attributes at switch tunnel and tunnel

we don't need, as i understand, 
SAI_SWITCH_TUNNEL_ATTR_TUNNEL_ENCAP_ECN_MODE
SAI_SWITCH_TUNNEL_ATTR_ENCAP_MAPPERS
SAI_SWITCH_TUNNEL_ATTR_TUNNEL_DECAP_ECN_MODE
SAI_SWITCH_TUNNEL_ATTR_DECAP_MAPPERS

these were already committed in previous commit, and should be cleaned imo
backporting should be minimal to what is required, and not to full objects

Reply:
check the change in related PR #1205 #1173 and #1206. 
remove the unnecessary attributes.
thanks!